src/lib/zkapp.ts

```diff
@@ -593,9 +593,9 @@ class SmartContract {
   address: PublicKey;
   tokenId: Field;
 
-  private _executionState: ExecutionState | undefined;
+  #executionState: ExecutionState | undefined;
```

Comment:
changes in this file are to work around some TS glitches when returning a `SmartContract` from a function (i.e., using a class factory)